### DISCUSSION TITLE
Is there a way to automatically launch OpenTrack in "active" mode?

### DISCUSSION THREAD

**Discussion by platypus65** [opentrack/opentrack#1647] [Q&A] [answered]
Is there a way to automatically launch OpenTrack in "active" mode? I know it's a small thing, but I hate hitting the "start" button after launching OpenTrack

**Comment by Pieloth** [comment, marked as answer]
Quick & dirty way that does the job:

In the "Game detection" option, add an item line (hit +)
Set the Executable field to: 
opentrack.exe 
And choose a profile, click Ok

Close opentrack and restart it.
Enjoy 😊

**Reply by platypus65** [reply]
Thank you so much! That works perfectly!
